src/utils/tools.ts

```diff
@@ -67,10 +67,14 @@ async function goRun(args: string): Promise<boolean> {
   const gobin = await configDir('bin')
 
   const env = { GOBIN: gobin, GOPATH: gopath, GO111MODULE: 'on' }
-  const cmd = `${formatCmd(env, `go ${args}`)}`
```

Comment:
Since `formatCmd()` isn't use anymore, could remove it from `tools.ts` and delete `tools.test.ts`?